src/util.cpp

```diff
@@ -246,6 +246,16 @@ std::string RemoveStringQuotes(const std::string &s) {
              : s;
 }
 
+std::string Trim(const std::string &s) {
+  const auto start =
```

Comment:
We have only 1 use for this function,  so seems all of this could be simplified by inlining it and making it specific to its use. Looks like all you want is to check for space after `///` ?

Reply:
True, most of the time I write ///<Space>comment and this leads to ugly description strings in the JSON schema like `" age"`

I inlined the Trim function, but I am not sure if this is more readable. With Trim as a function, we would have many benefits:
- The code gets a name, so the intent of the code is clear when reading the function "PrepareDescription". I tried to simulate this with a comment
- The compiler will (probably) inline this function anyway, so performance wise it will not have any advantages
- The code within Trim can be reused more easily.

I will leave it up  to you if you want to take aa659ac or 0db1631.

Reply:
Having to follow code thru a function call is a very costly way to give it an intent. This isn't about performance. Trying to predict the nature of future reuse that may never happen just creates unnecessary complexity.